docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc

```diff
@@ -925,7 +925,7 @@ public class ExampleSFSB {
 
 == Tips for using the latest Hibernate ORM
 
-http://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html[Read about using Hibernate ORM feature packs on WildFly].
+https://docs.jboss.org/hibernate/orm/5.4/topical/html_single/wildfly/Wildfly.html[Read about using Hibernate ORM feature packs on WildFly].
```

Comment:
I believe L926-928 should just be removed.  The linked doc is about the pre-Galleon feature packs, and WildFly hasn't provided those for quite a long time now, so what the doc says no longer works. I assume that is why Hibernate removed that doc from their 6.x docs.

@scottmarlow FYI.

Reply:
Thanks for your feedback, I'm preparing a fix and remove the link.

Reply:
Thanks Boris for noticing the Hibernate ORM support for adding new ORM releases on WildFly.  I agree with Brian as that was later removed from the Hibernate ORM project.